src/services/api.service.js

```diff
@@ -1,8 +1,8 @@
 const serverLatency = 0;
 
-export function get (endpoint, payload = {}) {
```

Comment:
worth adding a .md file to explain the api layer / how endpoints work ect. ?

Reply:
I'll do this in another branch, as I'm planning to add more endpoints